bin/configure-tower-projects.py

```diff
@@ -640,15 +679,101 @@ def add_member(self, user: str) -> dict:
         self.members[user] = member
         return member
 
+    def add_member_to_team(self, team_id: int, user: str) -> int:
+        """Add user to given team within an organization
+
+        Args:
+            team_id (int): Team identifier
+            user (str): Email address for the user
+
+        Returns:
+            int: Team member identifier, which is the same as the
+                organization member identifier
+        """
+        endpoint = f"/orgs/{self.id}/teams/{team_id}/members"
+        data = {"userNameOrEmail": user}
+        response = self.tower.request("POST", endpoint, json=data)
+        # If the user is already a member, you get the following message:
+        #   "The member is already associated with the team"
+        # If this happens, just retrieve the member ID from the organization
+        if "message" in response and "already" in response["message"]:
+            member = self.add_member(user)
+            member_id = member["memberId"]
+        else:
+            member_id = response["member"]["memberId"]
+        return member_id
+
+    def remove_member_from_team(self, team_id: int, member_id: int) -> Dict:
+        """Remove a member from an organization team
+
+        Args:
+            team_id (int): Team identifier
+            member_id (int): Member identifier
+        """
+        endpoint = f"/orgs/{self.id}/teams/{team_id}/members/{member_id}/delete"
+        response = self.tower.request("DELETE", endpoint)
+        return response
+
+    def create_team(self, team_name: str) -> int:
+        """Create team under organization with the given name
+
+        Args:
+            team_name (str): Team name
+
+        Returns:
+            int: Team identifier
+        """
+        # Check if the team already exists
+        endpoint = f"/orgs/{self.id}/teams"
+        response = self.tower.request("GET", endpoint)
+        for team in response["teams"]:
+            if team["name"] == team_name:
+                return team["teamId"]
+        # If team doesn't exist, create one
+        data = {"team": {"name": team_name, "description": None, "avatar": None}}
+        response = self.tower.request("POST", endpoint, json=data)
+        return response["team"]["teamId"]
+
+    def list_team_members(self, team_id: int) -> List[int]:
+        """Retrieve a list of team member IDs
+
+        Args:
+            team_id (int): Team identifier
+
+        Returns:
+            List[int]: List of team member IDs
+        """
+        endpoint = f"/orgs/{self.id}/teams/{team_id}/members"
+        response = self.tower.request("GET", endpoint)
+        team_member_ids = list()
+        for team_member in response["members"]:
+            team_member_ids.append(team_member["memberId"])
```

Comment:
Nit: use list comprehension
```suggestion
        team_member_ids = [
            team_member["memberId"]
            for team_member in response["members"]
        ]
```